rqt_image_view/src/rqt_image_view/image_view.cpp

```diff
@@ -77,11 +77,20 @@ void ImageView::initPlugin(qt_gui_cpp::PluginContext& context)
 
   ui_.save_as_image_push_button->setIcon(QIcon::fromTheme("image"));
   connect(ui_.save_as_image_push_button, SIGNAL(pressed()), this, SLOT(saveImage()));
+
+  pub_topic_custom_ = false;
```

Comment:
Agreed, I just adapted the #51 code to the latest version.

I'll apply now a few fixes, I will remove entirely pub_topic_custom_ since I believe it's just confusing.